src/SDKs/Azure.Base/data-plane/Azure.Base.Tests/PipelineTests.cs

```diff
@@ -18,11 +18,11 @@ public class PipelineTests
         string expected = @"ProcessingRequest : Get https://contoso.a.io/ # ErrorResponse : 500 # ProcessingResponse : Get https://contoso.a.io/ # ProcessingRequest : Get https://contoso.a.io/ # ProcessingResponse : Get https://contoso.a.io/";
 
         [Test]
-        public void Basics() {
-
-            var options = new HttpPipelineOptions(new MockTransport(500, 1));
-            options.RetryPolicy = new CustomRetryPolicy();
-            options.LoggingPolicy = new LoggingPolicy();
+        public void Basics()
+        {
+            var options = new HttpPipelineOptions()
```

Comment:
I am not a big fan of fluent APIs. Can we start with:
```c#
var options = new HttpPipelineOptions();
options.AddLogging();
options.Append(...);
...    
 
```

Reply:
Sure